conans/client/grapher.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans.model.ref import ConanFileReference
+
+class ConanGrapher(object):
+    def __init__(self, project_reference, deps_graph):
+        self._deps_graph = deps_graph
+        self._project_reference = project_reference
+
+    def graph(self):
```

Comment:
Sorry, I misunderstood what you meant. I've updated the PR to print to a file named `graph.dot` if no argument is supplied, and to a file with a name matching the supplied argument if given. The tests have been updated as well. Thanks for the feedback!